### DISCUSSION TITLE
[bug] Can not jump to unresolved conversation if file is not expanded

### DISCUSSION THREAD

**Discussion by KarlBlomquist** [community/community#8531] [Pull Requests]
If I click on "Files changed" / "Conversations" / "Jump to conversation" / "Unresolved conversations"  one of my unresolved conversations is within a large diff that by default is not expended to be shown on the current page.
When I click on that unresolved conversation in the large diff nothing happens.   No error, no change in page, no jump to conversation.
If I scroll down and expand the large diff file in question, then clicking on the unresolved conversation will work - but this is not at all efficient.

**Comment by CalvinWilkinson** [comment]
I am currently having the same issue!!  I swear this maybe did this before?

But yes, if all of the file changes are collapsed, clicking any unresolved conversation refreshes the page but the file that the conversation is contained in does not expand or anything.

I feel like we should have a filter of some kind that filters out files with unresolved comments, resolved comments, no comments, etc.

Also a button that with one click could expand or collapse all files would be fantastic!!

**Reply by CalvinWilkinson** [reply]
I retract what I said about expanding or collapsing all files in the **Files Changed** section of a PR.

I just learned you can just use the `Alt + Click` of the mouse on the expand or collapse icon on any file header and this works!!

🤭

**Comment by markfickett** [comment]
I just ran into this. The current behavior is very confusing, I assumed the "jump to conversation" button was just broken. Thanks for asking this Q since it pointed me to the workaround. Better behavior would be to auto-expand the file when clicking to jump to the conversation.
